turbopack/crates/turbo-tasks/src/capture_future.rs

```diff
@@ -64,8 +70,22 @@ pub fn add_allocation_info(alloc_info: AllocationInfo) {
     });
 }
 
+#[derive(Debug, Clone)]
+pub struct TurboTasksPanic {
+    pub message: TurboTasksExecutionErrorMessage,
+    pub location: Option<String>,
+}
+
+impl Display for TurboTasksPanic {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.message)
+    }
+}
+
+impl std::error::Error for TurboTasksPanic {}
```

Comment:
See my previous comment about using this as a source...

If you do decide to return this as a source from `TurboTasksExecutionError`, then this should be an `Error`. If you don't do that and you just print the message in `TurboTasksExecutionError`, then this probably doesn't need to a real `Error` type.